cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,66 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
-
-  it('should have the ability to assert automatically resolved alerts', () => {
-
+  it('Click on the first button and assert the text inside the alert.', () => {
+    cy.contains('button', 'Click me').click();
+    cy.on('window:confirm', () => true);
   });
 
-  it('should have the ability to assert scheduled allert', () => {
-
+  it('Click on the second button assert the' +
+   'text inside the alert is shown in 5 secons.', () => {
+    cy.get('button[id = "timerAlertButton"]').click();
```

Comment:
There is a shorter way to make a selector here (check how to make a selector using id). Please fix everywhere